content_dev/docs/tutorials/dess/rpi/7-activation/index.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: codelab
+
+title: 'Activation of @sign' # Step Name
```

Comment:
branding
'@sign' --> 'atSign'

Reply:
Thank you @JeremyTubongbanua!! I have made the rebranding and SEO description modifications 😃